### PR TITLE
Added detailed code on how to implement aws s3 and firebase with image picker. 

### PR DESCRIPTION
**Pull request by evanserickson**
…o added detailed instructions on firebase as well.

# Why

# How

# Test Plan

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by brentvatne — CHANGES_REQUESTED** [review]
i think we should not have this information on this page but instead link out to the example in [expo/examples](http://github.com/expo/examples). we should also link out to the other examples we have in that repo for integrating with a node backend and with firebase :)

for the additional context in this example (like about `amplify configure`) we should add it to the readme in the example repo

**Comment by EvanErickson** [comment]
Sounds great! I will make the changes soon and re push.

**Comment by EvanErickson** [comment]
I added examples

File: docs/pages/versions/unversioned/sdk/imagepicker.md [hunk 1]
```diff
@@ -82,6 +82,15 @@ When you run this example and pick an image, you will see the image that you pic
 }
 ```
 
+## Using ImagePicker with AWS S3
+
+Please refer to our example [here](https://github.com/expo/examples/tree/master/with-aws-storage-upload). Make sure you follow Amplify's docs to set your project up correctly.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/imagepicker.md:87]:
```suggestion
Please refer to the [with-aws-storage-upload example](https://github.com/expo/examples/tree/master/with-aws-storage-upload). Follow [Amplify docs](https://docs.amplify.aws/) to set your project up correctly.
```

File: docs/pages/versions/unversioned/sdk/imagepicker.md [hunk 2]
```diff
@@ -82,6 +82,15 @@ When you run this example and pick an image, you will see the image that you pic
...
+
+
+## Using ImagePicker with Firebase
+
+Please refer to our example [here](https://github.com/expo/examples/tree/master/with-firebase-storage-upload). Make sure you follow Firebases' docs to set your project up correctly.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/imagepicker.md:92]:
```suggestion
Please refer to the [with-firebase-storage-upload example](https://github.com/expo/examples/tree/master/with-firebase-storage-upload). Make sure you follow the ["Using Firebase"](/guides/guides/using-firebase.md) docs to set your project up correctly.
```

**Review by brentvatne — APPROVED** [review]

**Pull request merged**